tutorials/09-similarity-sesarch-for-motif-mining.ipynb

```diff
@@ -0,0 +1,724 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    print(response)
</pre><p>Change it to response.as_df()</p><p>same for other cellls</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/georgia-tech-db/eva/pull/700/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>